spec/factories/users.rb

```diff
@@ -1,38 +1,21 @@
 # frozen_string_literal: true
 FactoryGirl.define do
-  factory :user, class: 'User' do
+  # this factory represents a "typical" LOA 1 user, in that, these
+  # are the attributes ID.me will return for LOA 1
+  factory :loa1_user, class: 'User' do
```

Comment:
possibly consider keeping the factory name as user, and making loa1_user and loa2_user variations on the generic user that just set the specific attribute.